desc/transform.py

```diff
@@ -43,7 +43,16 @@ class Transform(IOAble):
     """
 
     _io_attrs_ = ["_grid", "_basis", "_derivatives", "_rcond", "_method"]
-    _static_attrs = ["_derivatives"]
+    _static_attrs = [
+        "_derivatives",
+        "_method",
+        "_rcond",
+        "_built",
+        "_built_pinv",
+        "num_n_modes",
+        "num_lm_modes",
```

Comment:
why must these be static but `num_zeta` etc are not static for `grid`? those are defined as the size of an array, which is also what `num_lm_modes` is defined as

Reply:
`num_zeta` isn't actually stored as an attribute of `Grid`, its just a property that dynamically grabs `grid.unique_zeta_idx.size`, and `unique_zeta_idx` is marked static